dmscripts/export_framework_applicant_details.py

```diff
@@ -238,6 +244,23 @@ def _create_row(framework_slug, record, count_statuses, framework_lot_slugs, dry
         _format_field(field, record["declaration"].get(field, ""))
         for field in DECLARATION_FIELDS[framework_slug]
     )
+
+    # For regenerating framework agreements with updated information (instead of the declaration details)
+    if include_central_supplier_details:
+        row.update((
+            ("supplier_registered_name", record["supplier"]["registeredName"]),
+            (
+                "supplier_registration_number",
+                record["supplier"].get(
+                    'companiesHouseNumber',
+                    record["supplier"].get('otherCompanyRegistrationNumber', "")
+                )
+            ),
+            ("supplier_contact_address1", record["supplier"]["contactInformation"][0]['address1']),
+            ("supplier_contact_city", record["supplier"]["contactInformation"][0]['city']),
+            ("supplier_contact_postcode", record["supplier"]["contactInformation"][0]['postcode']),
+        ))
```

Comment:
I personally feel this would be more readable as a dict than as an tuple of tuples, feel free to disagree though.

Reply:
Broadly agree. Originally it was in the existing tuple above, and I moved it out. Happy to convert both to a dict...

Reply:
That would be 🔥

Reply:
Also realised that `get_csv_rows` also returns the field headers, which I... haven't updated. Going to add those now.